README.md

```diff
@@ -75,6 +75,12 @@ In this sense, they represent asynchronous functions running in the background o
 
 Standard transformation strategies: *celery/remote*
 
+The tranformation strategy has consolidated the execution of the
+transformation with the get() method to unify the strategy interfaces.
+get() is intended to start an asyncronous process and return a
+*tast_id* which can be queried using the status() method (outside of a
```

Comment:
```suggestion
transformation with the `get()` method to unify the strategy interfaces.
`get()` is intended to start an asynchronous process and return a
*task_id* which can be queried using the `status()` method (outside of a
```

I suppose we're not doing one-line-per-one-sentence in README files anymore (hate that), so I won't suggest changes for that...